lib/elements/stroke.py

```diff
@@ -361,11 +368,23 @@ def reverse_rails(self):
            default=0,
            unit='mm',
            select_items=[('stroke_method', 'ripple_stitch')],
-           sort_index=16)
+           sort_index=23)
     @cache
     def grid_size(self):
         return abs(self.get_float_param("grid_size_mm", 0))
 
+    @property
+    @param('grid_first',
+           _('Stitch grid first'),
+           tooltip=_('Reverts the stitch paths, so that the grid will be stitched first'),
```

Comment:
I think "revert" is a bit confusing here, since it makes me think "undo".  How about "swap"?

Reply:
It is a lazy method and actually reverses the path...

Reply:
Oh, I see.  Hmm.  Maybe "reverse", then?